packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/sdk/test/arm-test/src/models/models.ts

```diff
@@ -17,7 +19,7 @@ export function _operationListResultDeserializer(
     nextLink: item["nextLink"],
   };
 }
-
+/* eslint-enable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
So many eslint-enable and eslint-disable lines, i am not sure we want to delievery these code lines. So ugly for me.